frontend/src/scenes/cohorts/cohortLogic.ts

```diff
@@ -12,7 +12,10 @@ import { router } from 'kea-router'
 
```

Comment:
@alexkim205 is changing this up in the new PR so up to him if he wants this merged or just to handle it in his work!